week-2/I-array-properties/exercise.js

```diff
@@ -6,7 +6,9 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length === 0) return true;
```

Comment:
Try and refactor this - `arr.length === 0)` already returns a boolean, so you don't really need the `if` statement ;)